package.json

```diff
@@ -4,20 +4,25 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon server.js",
-    "lint": "eslint ."
+    "start": "node server.js",
+    "server": "nodemon server.js",
+    "lint": "eslint .",
+    "client": "npm run start --prefix ui",
+    "dev": "concurrently \"npm run server\" \"npm run client\""
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "bcrypt": "^4.0.1",
```

Comment:
Use eslint code linter with default google rules to validate your code;
Use Joi to validate all incoming requests structure;

Some of requirements to task